doc/src/manual/strings.md

```diff
@@ -371,6 +400,34 @@ julia> string(greet, ", ", whom, ".\n")
 "Hello, world.\n"
 ```
 
+A situation which is important to be aware of is when invalid UTF-8 strings are concatenated.
+In that case string may contain different characters than those that constitute concatenated
```

Comment:
"the resulting string" and "that constitute input strings"?

Below, typo "sting". "such a string" could just be "its [number of characters]".